packages/create-block/lib/templates.js

```diff
@@ -94,13 +92,13 @@ const getPrompts = ( templateName ) => {
 	} );
 };
 
-const hasWPScriptsEnabled = ( templateName ) => {
-	return getTemplate( templateName ).wpScriptsEnabled || false;
+const hasWPScriptsEnabled = ( blockTemplate ) => {
+	return ! ( blockTemplate.wpScripts === false );
```

Comment:
Perhaps more expressively as

```suggestion
	return blockTemplate.wpScripts !== false;
```